lake/top/lake_top.py

```diff
@@ -1076,7 +1076,7 @@ def get_static_bitstream(self,
         # handling arrays in the interface.
         # To alleviate this, we create the flattened wrapper so we can query widths of config
         # registers and trim values to their bitwidths...
-        print(f"Current_name: {self.name}")
+        # print(f"Current_name: {self.name}")
```

Comment:
Can just nuke this, was only for debugging I believe.

Reply:
Deleted.